lyricsheets/models/karaoke.py

```diff
@@ -118,6 +233,51 @@ def calculate_char_fade_offsets(self, style: Style, transitionDuration: timedelt
             char.fadeOffset = charAccTime
             charAccTime += lineCharTime
 
+    def calculate_sizing(self, style: Style):
+        fontScaler = FontScaler(style)
+
+        self.width = fontScaler.get_length(self.text)
+        self.height = style.fontSize
+
+        for syl in self.kara:
+            syl.calculate_sizing(style)
+
+    def calculate_positions(self, style: Style, resX: int = 1920, resY: int = 1080):
+        # Horizontal positioning
+        if style.alignment.value % 3 == 1:
```

Comment:
Just make it explicit and do `style.alignment in (..., ..., ...)`

Reply:
fixed